modules/installation-configuration-parameters.adoc

```diff
@@ -1042,6 +1042,54 @@ within link:https://azure.microsoft.com/en-us/global-infrastructure/regions[a re
 |The availability zones where the installation program creates compute machines.
 |String list
 
+|`compute.platform.azure.settings.securityType`
+|Enables confidential VMs or trusted launch for compute nodes. This option is not enabled by default.
+|`ConfidentialVM` or `TrustedLaunch`.
+
+|`compute.platform.azure.settings.confidentialVM.uefiSettings.secureBoot`
+|Enables secure boot on compute nodes if you are using confidential VMs.
+|`Enabled` or `Disabled`. The default is `Disabled`.
+
+|`compute.platform.azure.settings.confidentialVM.uefiSettings.virtualizedTrustedPlatformModule`
+|Enables the virtualized Trusted Platform Module (vTPM) feature on compute nodes if you are using confidential VMs.
+|`Enabled` or `Disabled`. The default is `Disabled`.
+
+|`compute.platform.azure.settings.trustedLaunch.uefiSettings.secureBoot`
+|Enables secure boot on compute nodes if you are using trusted launch.
+|`Enabled` or `Disabled`. The default is `Disabled`.
+
+|`compute.platform.azure.settings.trustedLaunch.uefiSettings.virtualizedTrustedPlatformModule`
+|Enables the vTPM feature on compute nodes if you are using trusted launch.
+|`Enabled` or `Disabled`. The default is `Disabled`.
+
+|`compute.platform.azure.settings.osDisk.securityProfile.securityEncryptionType`
```

Comment:
`compute.platform.azure.settings.osDisk.securityProfile.securityEncryptionType`
=> 
`compute.platform.azure.osDisk.securityProfile.securityEncryptionType`

similar issues on `controlPlane` and `defaultMachinePlatform`

Reply:
another comment here is that this parameter can be set only for Confidential VMs.
```
$ ./openshift-install explain installconfig.platform.azure.defaultMachinePlatform.osDisk.securityProfile.securityEncryptionType
KIND:     InstallConfig
VERSION:  v1

RESOURCE: <string>
  Valid Values: "VMGuestStateOnly","DiskWithVMGuestState"
  SecurityEncryptionType specifies the encryption type of the managed disk. It is set to DiskWithVMGuestState to encrypt the managed disk along with the VMGuestState blob, and to VMGuestStateOnly to encrypt the VMGuestState blob only. When set to VMGuestStateOnly, the VTpmEnabled should be set to true. When set to DiskWithVMGuestState, both SecureBootEnabled and VTpmEnabled should be set to true. It can be set only for Confidential VMs.
```

Reply:
Good catch, thanks